File: app/src/main/java/com/example/fragmentexample/MainActivity.java
package com.example.fragmentexample;

import android.os.Bundle;
import android.os.PersistableBundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

public class MainActivity extends AppCompatActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        Fragment1 fragment1 = new Fragment1();
        fragment1.textTitle("Hello");

        FragmentTransaction ft1 = getSupportFragmentManager().beginTransaction();
        ft1.add(R.id.frame1, fragment1);
        ft1.commitAllowingStateLoss();


        Fragment2 fragment2 = new Fragment2();
        fragment2.textTitle("Bukunmi");

        FragmentTransaction ft2 = getSupportFragmentManager().beginTransaction();;
        ft2.add(R.id.frame2, fragment2);
        ft2.commitAllowingStateLoss();
    }
}

File: app/src/main/java/com/example/fragmentexample/MainActivity.kt
//package com.example.fragmentexample
//
//import androidx.appcompat.app.AppCompatActivity
//import android.os.Bundle
//import androidx.fragment.app.FragmentTransaction
//
//class MainActivity : AppCompatActivity() {
//
//    override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)
//
//
//        val fragment1 = Fragment1()
//        fragment1.textTitle = "Hello"
//
//        val ft1: FragmentTransaction = supportFragmentManager.beginTransaction()
//        ft1.add(R.id.frame1, fragment1)
//        ft1.commitAllowingStateLoss()
//
//
//        val fragment2 = Fragment2()
//        fragment2.textTitle = "Hello"
//
//        val ft2: FragmentTransaction = supportFragmentManager.beginTransaction()
//        ft2.add(R.id.frame2, fragment2)
//        ft2.commitAllowingStateLoss()
//
//    }
//}
